clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -694,16 +694,18 @@ describe("Consent i18n", () => {
       });
     });
 
-    // TODO (PROD-1598): enable this test and add other cases as needed!
-    describe.skip("when user selects their own locale", () => {
+    describe("when user selects their own locale", () => {
       it(`localizes in the user selected locale (${SPANISH_LOCALE})`, () => {
         // Visit the demo site in English, but expect Spanish translations when the user selects
         visitDemoWithI18n({
           navigatorLanguage: ENGLISH_LOCALE,
           globalPrivacyControl: true,
           fixture: "experience_banner_modal.json",
         });
-        // TODO (PROD-1598): select Spanish from banner
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(200); // wait for the overlay to appear
```

Comment:
nit: I agree with eslint! You should trust Cypress to run-and-retry as much as possible!

Reply:
generally I do too, but a) saw this used in other places where our cypress tests need to wait for the banner so assumed it was an acceptable exception. b) Cypress was running but not re-trying this, just times out and fails. (even waiting 100ms isn't long enough). Open to other solutions.